.github/workflows/CI.yml

```diff
@@ -230,7 +230,7 @@ jobs:
         run: dotnet fsi scripts/inconsistentNugetVersionsInDotNetProjectsAndFSharpScripts.fsx
       - name: FSharpLint
         run: |
-          dotnet fsi scripts/runFSharpLint.fsx
+          dotnet fsi scripts/runFSharpLint.fsx conventions.sln
```

Comment:
@webwarrior-ws it's fine that we prepare runFSharpLint.fsx to receive an argument, but if not given arguments, it should find all solution files and all .fsx files on its own, let's do this in a new commit that also removes this argument

Reply:
Read my previous comment again, it doesn't contradict what I said before.

Reply:
(Take in account that in the sentence 2invoked with the solution path as an argument", I'm not talking about runFsharpLint.fsx argument, but FSharpLint console argument)

Reply:
Oh, I got what you mean. FSharpLint tool **is** invoked with solution as argument. But if .sol file is not passed to runFSharpLint.fsx, then the script tries to find solution file in the target dir, and then in fallback dir.